src/config/mail.ts

```diff
@@ -2,7 +2,7 @@ import { BullModuleOptions } from '@nestjs/bull';
 
 export const mailBullConfig: BullModuleOptions = {
   redis: {
-    host: 'localhost',
```

Comment:
We do not upload env variables to the repository, for the next time, be carefull